neps/nep-0452.md

```diff
@@ -0,0 +1,234 @@
+
+By definition, anyone with an access key can interact with the blockchain and since there is a private key embedded in the link, this removes the need for the end-user to have a wallet.
+
+## Motivation
+
+Linkdrops are an extremely powerful tool that enable seamless onboarding and instant crypto experiences with the click of a link. The original [near-linkdrop](https://github.com/near/near-linkdrop) contract provides a minimal interface allowing users to embed $NEAR within an access key and create a simple Web2 style link that can then be used as a means of onboarding. This simple $NEAR linkdrop is not enough as many artists, developers, event coordinators, and applications want to drop more digital assets such as NFTs, FTs, tickets etc.
+
+As linkdrop implementations start to push the boundaries of what’s possible, new data structures, methods, and interfaces are being developed. There needs to be a standard data model and interface put into place to ensure assets can be claimed independent of the contract they came from. If not, integrating any application with linkdrops will require customized solutions, which would become cumbersome for the developer and deteriorate the user onboarding experience. The linkdrop standard addresses these issues by providing a simple and extensible standard data model and interface.
+
+The initial discussion can be found [here](https://gov.near.org/t/official-linkdrop-standard/32463/1).
+
+## Specification
+
+### Example Scenarios
+
+_Pre-requisite Steps_: Linkdrop creation:
+The linkdrop creator that has an account and some $NEAR:
+
+- creates a keypair locally (`pubKey1`, `privKey1`). (The keypair is not written to chain at this time)
+- calls a method, passing the `pubKey1` and desired $NEAR amount as arguments, to a contract that implements the linkdrop standard.
+- The contract will map the `pubKey1` to the desired balance for the linkdrop.
+- The contract will then add the `pubKey1` as a function call access key with the ability to call `claim` and `create_account_and_claim`. This means that anyone with the `privKey1` (see above), can sign a transaction on behalf of this contract (signer id set to contract id) with a function call to call one of the mentioned functions to claim the assets.
+
+#### Claiming a linkdrop without a NEAR Account
+
+A user with _no_ wallet that is claiming the assets associated with an existing public key, already registered on the linkdrop contract:
+
+- generates a new keypair (`pubKey2`, `privKey2`) locally. (This new keypair is not written to chain)
+- chooses a new account ID such as benji.near.
+- calls `create_account_and_claim`. The transaction is signed on behalf of the linkdrop contract (`signer_id` is set to the contract address) using `privKey1`.
```

Comment:
```suggestion
- calls `create_account_and_claim`. The transaction is signed on behalf of the linkdrop contract (`signer_id` is set to the contract address) using `privKey1`. The `privKey1` is usually encoded in an dapp link -- dapp will extract the key and prepare a transaction:
```

Reply:
I don't like this suggestion as it's referencing an assumption that isn't always the case. Linkdrops can be claimed through the CLI and no on dApps. I'd like to keep this section as what will happen *everytime* which is environment agnostic (dApp vs. CLI etc.)

Reply:
I think it's good to make a context. However if think it's not needed then OK.